controlplane/kubeadm/main.go

```diff
@@ -193,6 +198,11 @@ func InitFlags(fs *pflag.FlagSet) {
 // Add RBAC for the authorized diagnostics endpoint.
 // +kubebuilder:rbac:groups=authentication.k8s.io,resources=tokenreviews,verbs=create
 // +kubebuilder:rbac:groups=authorization.k8s.io,resources=subjectaccessreviews,verbs=create
+// ADD RBAC for CRD Migrator.
```

Comment:
```suggestion
```

kind of duplicate, at least inconsistent when comparing main.go's

Reply:
Thx, yup that was not intentional